active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
Okay, thanks. Interval trees are used in a variety of contexts so having that lower-level data structure available in a separate package (with a generalized API) may be useful to the Python community if someone has time to implement and maintain that package.

I'm wondering if `Sequence.features` or `Sequence.feature_metadata` is a more intuitive/appropriate name for this type of metadata in scikit-bio, that way we're not implying a particular implementation (via interval tree). What do you think about these names? If there are other scikit-bio objects that would store this type of metadata in the future, is the name generalized enough to make sense in other contexts?